backends/platform/libretro/include/surface.libretro.h

```diff
@@ -0,0 +1,345 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef GRAPHICS_SURFACE_H
+#define GRAPHICS_SURFACE_H
+
+#include "common/scummsys.h"
+
+namespace Common {
+struct Rect;
+}
+
+#include "graphics/pixelformat.h"
+
+namespace Graphics {
+
+/**
+ * An arbitrary graphics surface, which can be the target (or source) of blit
+ * operations, font rendering, etc.
+ */
+struct Surface {
```

Comment:
What is the purpose of this thing? It looks like an override/substitute for `Graphics::Surface` struct

Reply:
This is coming from the legacy core, I guess the original intent was to expose [this](https://github.com/scummvm/scummvm/blob/60a40ed6f499c91657f3e22f278ed3feca556564/graphics/surface.h#L84-L88). I've removed the entire thing and replaced uses of `pixels` with `getPixels()`